synapse/cortex.py

```diff
@@ -580,6 +580,95 @@ def _onFeedTagDel(self, snap, mesg):
     # def _addSynUndo(self, snap, items):
         # TODO apply splices in reverse
 
+    def _addSynIngest(self, snap, items):
+        # Check and wrap items in a list in the event that
```

Comment:
This could be a problematic convention for other feed types ( whose items might be lists themselves ) and makes the API a bit inconstant.  Lets just require items to be a list of ingest dictionaries.